queries/djot/injections.scm

```diff
@@ -0,0 +1,22 @@
+(code_block
+  (language) @injection.language
+  (code) @injection.content)
+
+(raw_block
+  (raw_block_info
+    (language) @injection.language)
+  (content) @injection.content)
+
+; There's no "latex-math" I could find?
```

Comment:
There is none. You can only inject actual languages, and "math mode" is not a language. You could lobby the tree-sitter-latex devs to split their parser (analogously to markdown) so that math mode is a separate language injected into latex itself. Not sure they want to do that, as it complicates things and AFAIR the parser is used in the texlab language server.

Reply:
I'll remove that comment.